src/main/java/filter/expression/FilterExpression.java

```diff
@@ -28,6 +28,11 @@
 
     void applyTo(TurboIssue issue, IModel model) throws QualifierApplicationException;
 
+    // Walks the syntax tree to get all problems with the input which are not severe
+    // enough to cause an error.
+
```

Comment:
Hmm, because the other methods' comments in the interface are not javadoc. If we make this javadoc, shouldn't we make the others javadoc as well?